Reminder-App/Reminder-App/ReminderDetailEditDataSource.swift

```diff
@@ -9,63 +9,72 @@ import UIKit
 
 class ReminderDetailEditDataSource: NSObject {
     
+    typealias ReminderChangeAction = (Reminder) -> Void
+    
     enum ReminderSection: Int, CaseIterable {
-        
         case title
         case dueDate
         case note
         
         // MARK: Public Methods
         func cellIdentifier(for row: Int) -> String {
-            
             switch self {
             case .title:
                 return "EditTitleCell"
             case .dueDate:
                 return row == 0 ? "EditDateLabelCell" : "EditDateCell"
             case .note:
-                return "EditnoteCell"
+                return "EditNoteCell"
             }
         }
     
-    // MARK: Computed Properties
-    var displayText: String {
-        switch self {
-        case .title:
-            return "Title"
-        case .dueDate:
-            return "Date"
-        case .note:
-            return "note"
+        // MARK: Computed Properties
+        var displayText: String {
+            switch self {
+            case .title:
+                return "Title"
+            case .dueDate:
+                return "Date"
+            case .note:
+                return "note"
+            }
         }
-    }
     
-    var numRows: Int {
-        switch self {
-        case .title, .note:
-            return 1
-        case .dueDate:
-            return 2
+        var numRows: Int {
+            switch self {
+            case .title, .note:
+                return 1
+            case .dueDate:
+                return 2
+            }
         }
     }
+    
+    // MARK: Initializers
+    init(reminder: Reminder, changeAction: @escaping ReminderChangeAction) {
```

Comment:
ReminderChangeAction should be **reminderChangeAction** property/variables should be lowerCamel Case

Reply:
the type is not the property but a typealias (see at line number 12)